packages/api/source/models/AppModel.ts

```diff
@@ -4,6 +4,35 @@ import paginate from "mongoose-paginate-v2";
 import type { App } from "../types";
 import { appStatuses } from "../utils/constants";
 
+const googleAuthSchema = new Schema(
+    {
+        enabled: {
+            type: Boolean,
+            required: true,
+        },
+        clientId: {
+            type: String,
+            required: true,
+        },
+        secret: {
+            type: String,
+            required: true,
```

Comment:
You can remove the `required` from all three. The Joi schema in the controllers don't have required.

Reply:
But it is supposed to be required according to [jira ticket](https://hypertool.atlassian.net/browse/HT-54)

Reply:
So we have to add `required` to Joi schema then?